### DISCUSSION TITLE
Hacktoberfest 2022 - MindsDB fabulous prizes!

### DISCUSSION THREAD

**Discussion by C0staTin** [mindsdb/mindsdb#3106] [Announcements]
### Join MindsDB at [Hacktoberfest 2022](https://mindsdb.com/hacktoberfest/?utm_medium=community&utm_source=slack&utm_campaign=hacktoberfest2022)! 

We have a lot of [rewards](https://mindsdb.com/hacktoberfest/?utm_medium=community&utm_source=slack&utm_campaign=hacktoberfest2022) and [issues](https://github.com/mindsdb/mindsdb/issues?q=is%3Aissue+is%3Aopen+label%3Ahacktoberfest) for first-time developers, low-code, and no-code contributors! 

Contribute and [Win](https://mindsdb.com/hacktoberfest/?utm_medium=community&utm_source=slack&utm_campaign=hacktoberfest2022) an awesome [Razer Blade 15 Laptop](https://www.razer.com/hard-bundles/Razer-Blade-15-Bundle-V5-Essential/RZHB-220912-11), cool audio gadgets, cash prizes, SWAG and more!

![hacktoberfest_1a_1,91x1_1200x628](https://user-images.githubusercontent.com/12409467/193908791-1f82476b-388a-49a7-98dd-b7aaa15f68d8.jpg)

**Comment by Yuerh** [comment]
邮件已收到，谢谢。
